lib/dpl/provider/pages.rb

```diff
@@ -13,6 +13,7 @@ class Pages < Provider
         - project-name [optional, defaults to fqdn or repo slug]
         - email [optional, defaults to deploy@travis-ci.org]
         - name [optional, defaults to Deployment Bot]
+        - deployment-file [optional, defaults to true]
```

Comment:
I think this should default to false. Helps keep the .travis.yml cleaner. I don't think it makes sense in the first place to make the file, but for those who want it they can enable it.